lib/fpm/package/python.rb

```diff
@@ -136,7 +139,31 @@ def download_if_necessary(package, version=nil)
     else
       logger.debug("using pip", :pip => attributes[:python_pip])
       # TODO: Support older versions of pip
-      safesystem(attributes[:python_pip], "download", "--no-clean", "--no-deps", "--no-binary", ":all:", "-i", attributes[:python_pypi], "--build", target,  want_pkg)
+
+      setup_cmd = [
+        attributes[:python_pip],
+        "download",
+        "--no-clean",
+        "--no-deps",
+        "--no-binary",
+        ":all:",
+        "-i", attributes[:python_pypi],
+      ]
+
+      if attributes[:python_trusted_host]
+        setup_cmd += [
+          "--trusted-host",
+          attributes[:python_trusted_host],
+        ]
+      end
+
+      setup_cmd += [
+        "--build",
+        target,
+        want_pkg,
+      ]
+      
+      safesystem(setup_cmd*" ")
```

Comment:
safesystem() can take a list of arguments, so I'll make a small change after merging to use it that way.